test/cluster/mzcompose.py

```diff
@@ -3164,3 +3164,85 @@ def subscribe():
                     ), e
             for thread in threads:
                 assert not thread.is_alive(), f"Thread {thread.name} is still running"
+
+
+def workflow_test_refresh_mv_warmup_after_restart(
+    c: Composition, parser: WorkflowArgumentParser
+) -> None:
+    """
+    Regression test for https://github.com/MaterializeInc/materialize/issues/25279
+    Test that bootstrapping selects an as_of for an MV dataflow in a way that allows it to warm up before its next
+    refresh.
+    """
+
+    with c.override(
+        Materialized(
+            additional_system_parameter_defaults={
+                "enable_refresh_every_mvs": "true",
+            },
+        ),
+        Testdrive(no_reset=True),
+    ):
+        c.down(destroy_volumes=True)
+
+        c.up("materialized")
+        c.up("clusterd1")
+
+        # Create a new cluster
+        c.sql(
+            "ALTER SYSTEM SET enable_unmanaged_cluster_replicas = true;",
+            port=6877,
+            user="mz_system",
+        )
+        c.sql(
+            """
+            CREATE CLUSTER cluster1 SIZE '1'
+            """
+        )
+
+        c.up("testdrive", persistent=True)
```

Comment:
Nit: Moving that after the `c.up("materialized")` moves all the `up`s into on place and possibly gives the testdrive more time to start up while we are running the `CREATE CLUSTER` command.

Reply:
Done